egs/commonvoice/asr1/run.sh

```diff
@@ -101,12 +101,9 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
         utils/fix_data_dir.sh data/${x}
     done
 
-    for x in ${train_set} ${recog_set}; do
-      # Remove features with too long frames in training data
-      max_len=3000
-      remove_longshortdata.sh  --maxframes $max_len data/${x} data/${x}_temp
-      mv data/${x}_temp data/${x}
-    done
+    mv data/${train_set} data/${train_set}_org
+    max_len=3000
+    remove_longshortdata.sh  --maxframes $max_len data/${train_set}_org data/${train_set}
```

Comment:
Many thanks!
Yes, it's right to make `${recog_set}` as it is (we must not modify the test set), but it would be better to remove long utterances for `${train_dev}` for efficient training (validation scoring during training).
So, how about the following? 

```suggestion
    for x in ${train_set} ${train_dev}; do
        mv data/${x} data/${x}_org
        max_len=3000
        remove_longshortdata.sh  --maxframes ${max_len} data/${x}_org data/${x}
    done
```

Reply:
Sounds good!

Reply:
If you reflect it, I'll merge this PR.